src/app/components/Staking/Staking.tsx

```diff
@@ -153,14 +153,12 @@ export const Staking: React.FC<StakingProps> = ({
       return;
     }
     const nextBlockHeight = btcHeight + 1;
-    const { stakingCapHeight, stakingCapSat, confirmationDepth } =
-      paramWithCtx.currentVersion;
+    const { stakingCapHeight, stakingCapSat } = paramWithCtx.currentVersion;
     // Use height based cap than value based cap if it is set
     if (stakingCapHeight) {
       setOverflow({
         isHeightCap: true,
-        overTheCapRange:
-          nextBlockHeight >= stakingCapHeight + confirmationDepth,
+        overTheCapRange: nextBlockHeight >= stakingCapHeight,
```

Comment:
```suggestion
        overTheCapRange: nextBlockHeight > stakingCapHeight,
```

The `stakingCapHeight` is still eligible to receive delegations